osidb/mixins.py

```diff
@@ -234,6 +235,35 @@ class Meta:
         abstract = True
 
 
+class PendingFlawCommentMixinManager(models.Manager):
```

Comment:
This functionality is unnecessary. Every API call is resolved as atomic (https://github.com/RedHatProductSecurity/osidb/blob/master/config/settings_prod.py#L74 - other non-prod environments have the same setting) so there cannot be a situation when the request crashes half way (eg. when calling Bugzilla) and there is a dangling `FlawComment` left. Also no other parallel thread should see anything half way done in the DB because of the atomicity (should be handled by Django).

Reply:
TIL!